src/core/core.plugin.js

```diff
@@ -321,5 +321,6 @@ module.exports = function(Chart) {
 	 * @todo remove at version 3
 	 * @private
 	 */
-	Chart.PluginBase = helpers.inherits({});
+	Chart.PluginBase = function() {};
```

Comment:
I think `Chart.PluginBase = Chart.Element.extend({});` is more in line with the previous implementation because it calls `initialize` at construction time.